Preprocessing/LabelEncoder.ecl

```diff
@@ -89,7 +89,7 @@ EXPORT LabelEncoder := MODULE
   *   </pre>
   */
   EXPORT GetMapping(key) := FUNCTIONMACRO
-    IMPORT Preprocessing.Utils.LabelEncoder;
+    IMPORT ML_Core.Preprocessing.Utils.LabelEncoder;
```

Comment:
This needs to be a relative import - using the syntax that is relative to the place the macro was defined.

IMPORT #$.Utils.LabelEncoder

same for the other definitions below.

Reply:
Fixed. Please review. Thank you. @ghalliday 

P.S. related JIRA: ML-525, ML-518

Cc @RogerDev